blog/Announcing-2-2.md

```diff
@@ -0,0 +1,83 @@
+
+The [fable-compiler/fable-import](https://github.com/fable-compiler/fable-import/) repository also contains other bindings generated with ts2fable that suffered similar problems, even if the size wasn't so big. Because of this and after discussing with the main contributors to Fable ecosystem, we've decided to abandon the idea of having a single repository with multiple Fable bindings automatically generated, and instead encourage the community to create (and maintain) bindings that may start with ts2fable but also include another layer with more idiomatic F# code, and that don't need to start with the `Fable.Import.` prefix.
+
+This means we're phasing out the fable-import repository and given that the browser bindings are the most used in Fable projects, we've already ported them to [another repository](https://github.com/fable-compiler/fable-browser). This contains not only one but multiple packages, roughly corresponding to [the Web APIs](https://developer.mozilla.org/es/docs/Web/API). The guidelines to make the namespaces consistent across these packages are:
+
+- Interfaces are contained in the `Browser.Types` namespace in all packages.
+- Values, [like `window` or constructors like `FileReader`](https://github.com/fable-compiler/fable-browser/blob/439b42070c9ed6afc6ec18499f22f3c238c221ae/src/Dom/Browser.Dom.Api.fs#L14-L22), are put in a module with the same name as the package (e.g. Fable.Browser.Dom -> Browser.Dom). This module is decorated with `AutoOpen` so in general you only need to `open Browser` to access any of the values in the Fable.Browser packages. Only to prevent conflicts you should need to qualify the module name (e.g. `Browser.Dom.document`).
+
+The `fable-browser` repository doesn't contain all Web APIs yet. Please contribute them or, if you prefer, it's also ok to publish packages for the missing APIs from your own profile.
+
+## Fable.PowerPack
+
+Similar problems affected Fable.PowerPack. It was originally intended as an extended Fable.Core but it had become a mix of unrelated modules and many times it was difficult for maintainers to remember who had authored what.
+
+Following the reasoning above, we've decided to deprecate Fable.PowerPack and replace it with single-purpose packages. For now, [Fable.Promise](https://github.com/fable-compiler/fable-promise) and [Fable.Fetch](https://github.com/fable-compiler/fable-fetch) have been released (only v2 beta is compatible with Fable.Core 3). The API is identical to the modules in Fable.PowerPack except for these changes:
+
+- Similar to Fable.Browser, the `Fable.` prefix is removed from the actual module names. In most cases (e.g. to use `Promise.map`) you won't need to open anything.
+- `Fetch.Fetch_types` is renamed to `Fetch.Types`.
+- The fetch methods in auto-serialization (like `fetchAs`) have been removed, please combine fetching with [Thoth.Json](https://github.com/thoth-org/Thoth.Json) or [Fable.SimpleJson](https://github.com/Zaid-Ajaj/Fable.SimpleJson).
```

Comment:
@alfonsogarciacaro Perhaps we can add a link to the old implementation of `fetchAs` so people know take a look at it and understand how to write their own version:

https://github.com/fable-compiler/fable-powerpack/blob/2459ca3772b564b68ea02d9e323e14118fc4bf5b/src/Fetch.fs#L359